### DISCUSSION TITLE
Threat Actor  & Apply Calculated Severity

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [simonec73/threatsmanager#13] [Q&A] [answered]
I have been trying to compare the base template structure of the MSTM and Threat manager Studio and found the following difference
![image](https://user-images.githubusercontent.com/31912418/145159948-bf77c38d-24b1-49fb-9724-cdd0166631c3.png)
(Ability to Add multiple mitigation and additional threat actor).  Is my understanding correct ?

Is "threat actor" standalone in Threats Manager Studio ?  In the future will it be related to threat types ?

Saw this new feature in the latest build - Apply Calculated Severity.  Not able to find any documentation on it.  Can you give a high level overview of the feature ?

**Comment by simonec73** [comment, marked as answer]
Hi, thank you for your comments and your continued interest. I confirm those differences.
Threat Actors are associated with Threat Events through Scenarios. They are not really used, yet, and this is the reason why you can find them mostly in Pioneer mode. I’ve already developed support for Quantitative Risk Analysis in a way which is similars to FAIR. It’s not ready, yet, but it will be added soon enough. This will make use of the Scenarios.
Finally, the Calculated Severity is an important new feature introduced with v1.5.0 and designed to provide an easy way to get severity calculated automatically. 
I’ve started updating the documentation, therefore this feature will be described in a few weeks. I’m planning to fix the gap in December. Expect this to be available in early January if not earlier.
